map-api-react/src/components/LayerMap.tsx

```diff
@@ -186,7 +182,8 @@ const LayerMap: React.FC<LayerMapProps> = () => {
 
       const geoJsonLayer = L.geoJSON(geoJsonData, {
         pointToLayer: (feature, latlng) => {
-          const { icon, label } = feature.properties as { icon: L.Icon<L.IconOptions> | L.DivIcon; label: string };
+          const { sidc, label } = feature.properties;
+          const icon = getMil2525Icon(sidc);
```

Comment:
Without this change, I got
```
A non-serializable value was detected in the state, in the path: `mapApiGateway.loadLayerError`. Value: TypeError: Cannot read properties of undefined (reading 'createIcon')
    at NewClass._initIcon (chunk-PQZ3PNSY.js?v=144ca945:5081:36)
    at NewClass.onAdd (chunk-PQZ3PNSY.js?v=144ca945:5011:16)
    at NewClass._layerAdd (chunk-PQZ3PNSY.js?v=144ca945:4454:16)
    at NewClass.whenReady (chunk-PQZ3PNSY.js?v=144ca945:2981:22)
    at NewClass.addLayer (chunk-PQZ3PNSY.js?v=144ca945:4475:16)
    at NewClass.eachLayer (chunk-PQZ3PNSY.js?v=144ca945:4631:20)
    at NewClass.onAdd (chunk-PQZ3PNSY.js?v=144ca945:4617:16)
    at NewClass._layerAdd (chunk-PQZ3PNSY.js?v=144ca945:4454:16)
    at NewClass.whenReady (chunk-PQZ3PNSY.js?v=144ca945:2981:22)
    at NewClass.addLayer (chunk-PQZ3PNSY.js?v=144ca945:4475:16) 
Take a look at the reducer(s) handling this action type: mapApiGateway/setLoadLayerError.
(See https://redux.js.org/faq/organizing-state#can-i-put-functions-promises-or-other-non-serializable-items-in-my-store-state)
```

This change switches icon computation to after so that we don't have to serialize it into geojson